class/defaults.yml

```diff
@@ -7,4 +7,4 @@ parameters:
       secretname: metallb-memberlist
     configmap_name: metallb
     charts:
-      metallb: 0.1.23
+      metallb: 1.1.0
```

Comment:
Why don't we upgrade the chart to the latest version (`2.6.11`)?

Reply:
That will be the next step. But the v2.x changes much, so to have a clearer diff, I've decided to do a intermediate step.